test-bench-utils/types/sinks/nosqlInjection/dynamodb.d.ts

```diff
@@ -3,7 +3,7 @@ export function _aws_sdk_DynamoDB_DocumentClient_prototype_scan({ input }: {
 }, { safe, noop }?: {
     safe?: boolean;
     noop?: boolean;
-}): Promise<"NOOP" | import("aws-sdk/lib/request").PromiseResult<AWS.DynamoDB.DocumentClient.ScanOutput, AWS.AWSError>>;
+}): Promise<any>;
```

Comment:
see above, I don't see changes to the related file--i think these are an accident.